rfcs/20220815-v3.0-API-cleanup/README.md

```diff
@@ -0,0 +1,91 @@
+        * End user must update `layer_normalization_forward` and `layer_normalization_backward` classes with proper calls to primitive descriptors.
+    * [`batch_normalization_forward`](dnnl.hpp#L7561) and [`batch_normalization_backward`](dnnl.hpp#L7699) constructors relying on a single data descriptor instead of source and destination will be replaced with constructors providing separate source and destination memory descriptors.
+        * End user must update `batch_normalization_forward` and `batch_normalization_backward` classes with proper calls to primitive descriptors.
+    * [`shuffle_forward`](dnnl.hpp#L11687) and [`shuffle_backward`](dnnl.hpp#L11769) constructors relying on a single data descriptor instead of source and destination will be replaced with constructors providing separate source and destination memory descriptors.
+        * End user must update `shuffle_forward` and `shuffle_backward` classes with proper calls to primitive descriptors.
+    * [`eltwise_forward`](dnnl.hpp#L6656) and [`eltwise_backward`](dnnl.hpp#L6757) constructors relying on a single data descriptor instead of source and destination will be replaced with constructors providing separate source and destination memory descriptors.
+        * End user must update `eltwise_forward` and `eltwise_backward` classes with proper calls to primitive descriptors.
+    * [`prelu_forward`](dnnl.hpp#L12647) and [`prelu_backward`](dnnl.hpp#L12742) constructors relying on a single data descriptor instead of source and destination will be replaced with constructors providing separate source and destination memory descriptors.
+        * End user must update `prelu_forward` and `prelu_backward` classes with proper calls to primitive descriptors.
+    * [`lrn_forward`](dnnl.hpp#L6165) and [`lrn_backward`](dnnl.hpp#L6268) constructors relying on a single data descriptor instead of source and destination will be replaced with constructors providing separate source and destination memory descriptors.
+        * End user must update `lrn_forward` and `lrn_backward` classes with proper calls to primitive descriptors.
+
+* Primitive descriptors:
+    * C API [`dnnl_convolution_forward_desc_init`](dnnl.h#1512), [`dnnl_convolution_backward_data_desc_init`](dnnl.h#1589), and [`dnnl_convolution_backward_weights_desc_init`](dnnl.h#1663) calls will be removed. [`dnnl_dilated_convolution_forward_desc_init`](dnnl.h#1555), [`dnnl_dilated_convolution_backward_data_desc_init`](dnnl.h#1626), and [`dnnl_dilated_convolution_backward_weights_desc_init`](dnnl.h#1704) will be renamed to `dnnl_convolution_forward_desc_init`, `dnnl_convolution_backward_data_desc_init`, and `dnnl_convolution_backward_weights_desc_init` correspondently. Note that these API calls will be updated to have three more arguments according to [this RFC](https://github.com/oneapi-src/oneDNN/pull/1391).
```

Comment:
> Note that these API calls will be updated to have three more arguments according

(nit) 2 for forward (engine, attributes) and 3 for backward (engine, attributes and hint)

Reply:
Thanks, fixed to reflect it.